airflow/contrib/kubernetes/configmap.py

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+class ConfigMapObject:
```

Comment:
This class is 1) not useful, and 2) putting it in it's own file is overkill. 3) never used apart form a variable in the tests that is then immediately overwritten with a string.

Similarly for the SecretObject. What are these classes for?

Reply:
@ashb I've removed them. I was thinking to extend them in future for example for namespace etc. but I realised it doesn't matter in this context right now.